test/custom-message.js

```diff
@@ -56,9 +50,9 @@ testRule(rule, {
   ],
 });
 
-testRule(rule, {
+testRule({
+  skip: true,
```

Comment:
- [ ] [`jest-preset-stylelint`](https://github.com/stylelint/jest-preset-stylelint) doesn't reject for invalid config.
   https://github.com/stylelint/jest-preset-stylelint/issues/58